src/foam/nanos/crunch/AgentCapabilityJunctionDAOSummaryView.js

```diff
@@ -90,6 +91,12 @@ foam.CLASS({
   ],
 
   properties: [
+    {
+      name: 'backLabel',
```

Comment:
Why is this a property? Does it ever need to be configured other than for locale?

Reply:
Yes, this property is used in the parent DAOSummaryView to display a custom back label. We overwrite it in this subclass.